test/testutil.py

```diff
@@ -140,6 +140,20 @@ def create_filesystem_customizations(rootfs: str):
     ]
 
 
+def pull_container(container_ref, target_arch=""):
+    if target_arch == "":
+        target_arch = platform.machine()
+
+    if target_arch not in ["x86_64", "amd64", "aarch64", "arm64", "s390x", "ppc64le"]:
+        raise RuntimeError(f"unknown host arch: {target_arch}")
+
+    output = subprocess.run([
```

Comment:
(super nitpick) now that we use `check=True` and not using the `output` anymore we might as well remove the `output = ` bit :)

Reply:
The tests complained so I had to remove that :p